packages/search/src/search.tsx

```diff
@@ -291,6 +311,11 @@ const InnerSearch = (
 			blur();
 		}
 
+		if ( event.key === 'Enter' && searchMode === 'on-enter' ) {
+			bypassOnSearchDebounce( keyword );
+			onSearchChange?.( keyword );
+		}
```

Comment:
@vinimotaa are we intentionally only supporting `Enter` key presses as a way to submit a search?

Reply:
hey @lsl this is what we discussed - the pending decision is to add or not to add a "trigger" button to activate the search. I'd like to hear from you all your opinions. I know for mobile we don't need, this would only be for desktop

Reply:
Hey @vinimotaa. I liked we removed the "search while typing" feature because that normally made me anxious to type fast :)

Maybe because I was used to it, I'm now missing the auto search, at least when I complete typing my search term. If this is intentional, I would suggest adding a trigger button to make it explicit to the user they need to do something to start the search.

Reply:
Thanks for your feedback @paulopmt1!

Good point - on this case, what I have in mind for the desktop version is to have something like this;

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/97929097/162214176-312aaa98-ffe2-447d-8f52-6777992e6794.png">


we could incorporate a CTA within the search bar. This would be something I'd like to get some feedback from @SaxonF as well!

Reply:
@vinimotaa I also would think maybe replacing the search icon with the close button

Reply:
Replacing the icon once there is text would be pretty nice but I think we're already borderline implementing two search components in one, might be better to drop it to the left of the Search button, it should look ok.